File: server/src/main/java/com/e/shortform/domain/report/enums/ReportType.java
package com.e.shortform.domain.report.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public enum ReportType {

    INAPPROPRIATE("INAPPROPRIATE"),
    SEXUAL("SEXUAL"),
    VIOLENCE("VIOLENCE"),
    HATE_SPEECH("HATE_SPEECH"),
    SPAM("SPAM"),
    COPYRIGHT("COPYRIGHT"),
    FALSE_INFO("FALSE_INFO"),
    PRIVACY("PRIVACY"),
    ILLEGAL("ILLEGAL"),
    ETC("ETC");

    private final String value;

}

File: client/src/shared/constants/enums/ReportType.ts
/** 신고 이유 */
export enum ReportType {

    INAPPROPRIATE   = "INAPPROPRIATE",  // 부적절한 영상
    SEXUAL          = "SEXUAL",         // 선정적
    VIOLENCE        = "VIOLENCE",       // 폭행
    HATE_SPEECH     = "HATE_SPEECH",    // 욕설
    SPAM            = "SPAM",           // 스팸
    COPYRIGHT       = "COPYRIGHT",      // 저작권
    FALSE_INFO      = "FALSE_INFO",     // 가짜 뉴스 or 거짓 정보
    PRIVACY         = "PRIVACY",        // 개인정보 침해
    ILLEGAL         = "ILLEGAL",        // 불법적인
    ETC             = "ETC"             // 기타

}
